### ISSUE TITLE
Executing `mget` with `JedisCluster` using jedis 2.8.1 gives compilation error.  

### ISSUE THREAD

**Issue by sandeepd-bsb** [xetorthio/jedis#1292]
So, I am testing out the redis cluster with jedis 2.8.1. I am trying to do an `mget` test on my `JedisCluster` object. However, my code fails to compile.

Here's what I see when I try to compile my code:

   `[javac]   symbol:   method mget(String[])
    [javac]   location: variable cluster of type JedisCluster`

Other methods like `get` and `set` work just fine.

#### Jedis version:
2.8.1 / 2.8.0
#### Redis version:
3.2
#### Java version:
1.8

**Comment by marcosnils** [comment]
@sandeepd-bsb how are you calling `mget`?

**Comment by sandeepd-bsb** [comment]
Hi @marcosnils, I am calling `mget` on my `JedisCluster` object.
`List<String> values = cluster.mget(keys.toArray(new String[keys.size()]));`. Does this help?

**Comment by marcosnils** [comment]
@sandeepd-bsb this works for me in 2.8 
```java
 JedisCluster jc = new JedisCluster(new HostAndPort("localhost", 30001));
    jc.set("test", "pepe");
    List<String> keys = new ArrayList<String>();
    keys.add("test");
    List<String> result = jc.mget(keys.toArray(new String[keys.size()]));
    System.out.println(result);
```

**Comment by sandeepd-bsb** [comment]
@marcosnils hmm, feels weird. I guess there might be something wrong with my jedis jar. Earlier I was using a different constructor for instantiating my `JedisCluster` object - one that takes a `Set` of `HostAndPort`. Now, I changed it to just `HostAndPort` and it gives me this: 
`RedisClusterTest.java:28: error: incompatible types: HostAndPort cannot be converted to Set<HostAndPort>
    [javac]         JedisCluster cluster = new JedisCluster(new HostAndPort(host, port));`

this along with the one above which still persists. I am using the jar from this link: http://mvnrepository.com/artifact/redis.clients/jedis/2.8.1.

**Comment by sandeepd-bsb** [comment]
@marcosnils hmm, feels weird. I guess there's something wrong with my jedis jar (btw I am using it from this link: http://mvnrepository.com/artifact/redis.clients/jedis/2.8.1). Earlier, I was using a different constructor for instantiating my `JedisCluster` object - one that takes a `Set` of `HostAndPort`. I now changed it to `JedisCluster cluster = new JedisCluster(new HostAndPort(host, port))` and it gives me the following error:

`RedisClusterTest.java:27: error: incompatible types: HostAndPort cannot be converted to Set<HostAndPort>
    [javac]         JedisCluster cluster = new JedisCluster(new HostAndPort(host, port));`. Any suggestions?

**Comment by marcosnils** [comment]
@sandeepd-bsb that constructor is available in 2.8.1. Check your project configuration as it should work

**Comment by sandeepd-bsb** [comment]
@marcosnils Issue was with my project config. Sorry for bothering you guys! fixed now.
